Source/Border.js

```diff
@@ -50,7 +50,7 @@ export class Border extends WrappedObject {
   */
   set flatColor(value) {
     // TODO : Create internal overloader that routes color conversion based on type (hex, int etc)
-    var hexToNSColor = ColorHelper.colorFromHex(value);
+    var hexToNSColor = ColorHelper.hexToMSColor(value);
```

Comment:
Well that would be misleading as there is no "Color" type and the other
conversion methods handle the generic colour approaches. This explicitly
enables _Hex To Native_ color for when devs need an actual native
implementation. 

The framework may provide coverage for 90% of edge cases
but going in/out of the two layers should be simplified as much as possible
as well - as in where possible provide a bridge to remove deeper
dependencies.

There is NSColor and MSColor too btw